elasticsearch/elasticsearch.yml

```diff
@@ -11,7 +11,7 @@ cluster.name: media-service
 # Node names are generated dynamically on startup, so you're relieved
 # from configuring them manually. You can tie this node to a specific name:
 #
-# node.name: "Franz Kafka"
```

Comment:
Use hostname as name (this is the private hostname containing the private ip in AWS). This makes matching instances in a cluster to their AWS identity a bit easier.